index.html

```diff
@@ -5502,7 +5504,101 @@ <h2>Base Context</h2>
   }
 }
 </pre>
+      </section>
+
+      <section class="informative">
+        <h3>Differences between Contexts, Types, and CredentialSchemas</h3>
 
+        <p>
+The verifiable credential and verifiable presentation data models leverage a
+variety of underlying technologies including [JSON-LD] and [JSON-SCHEMA-2018].
+This section will provide a comparison of the <code>@context</code>,
+<code>type</code>, and <code>credentialSchema</code> properties, and cover some
+of the more specific use cases where it is possible to use these features of the
+data model.
+        </p>
+        
+        <p>
+The primary purpose of the <code>type</code> property is to uniquely
+identify the type of the <code>verifiableCredential</code> in which it
+appears i.e. to indicate which set of claims the
+<code>verifiableCredential</code> contains. This property is mandatory and
+the value "VerifiableCredential" must appear in the set of values. Whilst
+it is good practice to include one additional value depicting the unique
+subtype of this verifiableCredential, it is not forbidden to either omit
+additional type values or to include multiple type values in the array.
+Since many verifiers will request verifiableCredentials of a specific
+subtype, then omitting the subtype value may make it more difficult for
```

Comment:
avoid use of normative MAY language in non-normative sections.

Reply:
@David-Chadwick what's your take on this one? I believe this was your addition

Reply:
replace may with could

Reply:
removed "may" and replaced with "could". It was updated to "might" during feedback below, but I found that too close to "may" so I changed it to @David-Chadwick suggestion.